README.rst

```diff
@@ -34,7 +34,7 @@ minor release, corresponding image pull tags and their release notes:
 +-------------------------------------------------------+------------+--------------------------------------+---------------------------------------------------------------------------+------------------------------------------------------------------------+
 | `v1.8 <https://github.com/cilium/cilium/tree/v1.8>`__ | 2020-09-30 | ``docker.io/cilium/cilium:v1.8.4``   | `Release Notes <https://github.com/cilium/cilium/releases/tag/v1.8.4>`__  | `General Announcement <https://cilium.io/blog/2020/06/22/cilium-18>`__ |
 +-------------------------------------------------------+------------+--------------------------------------+---------------------------------------------------------------------------+------------------------------------------------------------------------+
-| `v1.7 <https://github.com/cilium/cilium/tree/v1.7>`__ | 2020-09-30 | ``docker.io/cilium/cilium:v1.7.10``   | `Release Notes <https://github.com/cilium/cilium/releases/tag/v1.7.10>`__  | `General Announcement <https://cilium.io/blog/2020/02/18/cilium-17>`__ |
+| `v1.7 <https://github.com/cilium/cilium/tree/v1.7>`__ | 2020-09-30 | ``docker.io/cilium/cilium:v1.7.10``  | `Release Notes <https://github.com/cilium/cilium/releases/tag/v1.7.10>`__  | `General Announcement <https://cilium.io/blog/2020/02/18/cilium-17>`__|
```

Comment:
I think there's one too many space here, because it doesn't look right:
![image](https://user-images.githubusercontent.com/1764210/94775152-9649fc00-03bf-11eb-9f3c-422cd70df91a.png)
```suggestion
| `v1.7 <https://github.com/cilium/cilium/tree/v1.7>`__ | 2020-09-30 | ``docker.io/cilium/cilium:v1.7.10``  | `Release Notes <https://github.com/cilium/cilium/releases/tag/v1.7.10>`__ | `General Announcement <https://cilium.io/blog/2020/02/18/cilium-17>`__|
```

Reply:
:face_with_head_bandage: Nice catch, easy to miss when the line doesn't fit in the width of my terminal window...

All the more reason to figure out some linter or script fix in #13366.